easybuild/easyconfigs/u/Unicycler/Unicycler-0.4.4-foss-2016b-Python-3.5.2.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'PythonPackage'
+
+name = 'Unicycler'
+version = '0.4.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/rrwick/Unicycler/'
+description = """hybrid assembly pipeline for bacterial genomes"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://github.com/rrwick/Unicycler/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['e25e7423119aec5e574324c403f8ae36d93c0ce1497853d42c1a49a2a924a1f4']
+
+dependencies = [
+    ('Python', '3.5.2'),
+    ('Bowtie2', '2.3.2'),
+    ('SAMtools', '1.3.1'),
+    ('Pilon', '1.22', '-Java-1.8.0_144', True),
```

Comment:
Should probably use -Java-1.8 as dependency instead and make use of the Java version wrapper.